src/components/navigation/Accordion.tsx

```diff
@@ -72,9 +72,15 @@ const AccordionItem = React.forwardRef<HTMLDivElement, AccordionItemProps>(funct
 				</Padding>
 			)}
 			{item.label && (
-				<StyledText size="medium" {...item.textProps}>
-					{item.label}
-				</StyledText>
+				<Tooltip label={item.label} overflowTooltip>
+					<Text
+						size="medium"
+						style={{ minWidth: 0, flexBasis: 0, flexGrow: 1 }}
```

Comment:
I can't understand why this style is moved from the styled component to an inline style. I think that by using a styled component is easier to potentially override the style with a class applied to the AccordionItem, while with the inline style there is no way to access the style of this Text. But I could surely be missing something.